frontend/packages/ceph-storage-plugin/src/components/modals/add-capacity-modal/add-capacity-modal.tsx

```diff
@@ -0,0 +1,141 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { RequestSizeInput, withHandlePromise } from '@console/internal/components/utils/index';
+import { StorageClassDropdown } from '@console/internal/components/utils/storage-class-dropdown';
+import {
+  createModalLauncher,
+  ModalTitle,
+  ModalBody,
+  ModalSubmitFooter,
+} from '@console/internal/components/factory';
+import './_add-capacity-modal.scss';
+import { k8sUpdate } from '@console/internal/module/k8s';
+
+export const AddCapacityModal = withHandlePromise((props: AddCapacityModalProps) => {
+  const { kind, obj, close, cancel } = props;
+  const dropdownUnits = {
+    TiB: 'TiB',
+    PiB: 'PiB',
+  };
+  const [requestSizeUnit, setRequestSizeUnit] = React.useState(dropdownUnits.TiB);
+  const [requestSizeValue, setRequestSizeValue] = React.useState('');
+  const [storageClass, setStorageClass] = React.useState('');
+  const [inProgress, setProgress] = React.useState(false);
+  const [errorMessage, setError] = React.useState('');
+  console.log(obj);
+  const getPresentValue = () => {
+    const currentValue = obj.spec.dataDeviceSet.capacity.split(' ');
+    const units = ['TiB', 'PiB'];
+    let value: any;
+    if (requestSizeUnit === units[0]) {
+      switch (currentValue[1]) {
+        case units[0]:
+          value = parseInt(currentValue[0], 10);
+          break;
+        case units[1]:
+          value = parseInt(currentValue[0], 10) * 1024;
+          break;
+        default:
+          value = 0;
+      }
+    } else if (requestSizeUnit === units[1]) {
+      switch (currentValue[1]) {
+        case units[0]:
+          value = parseInt(currentValue[0], 10) / 1024;
+          break;
+        case units[1]:
+          value = parseInt(currentValue[0], 10);
+          break;
+        default:
+          value = 0;
+      }
+    }
+
```

Comment:
Nit - remove this empty line

Reply:
> @a2batic is the alignment for the `Used` next to requested capacity correct?
> 
> @yuvalgalanti ^^

I think the fields should be 

> @a2batic is the alignment for the `Used` next to requested capacity correct?
> 
> @yuvalgalanti ^^

see the attached design. the text area is bigger and the dropdown is smaller
![image](https://user-images.githubusercontent.com/44394257/63522830-bb541600-c501-11e9-9978-52afa10bc46e.png)

Reply:
ack